datajoint/connection.py

```diff
@@ -17,7 +17,7 @@
 logger = logging.getLogger(__name__)
 
 
-def conn(host=None, user=None, password=None, init_fun=None, reset=False):
+def conn(host=None, user=None, password=None, init_fun=None, ssl=True, reset=False):
```

Comment:
```suggestion
def conn(*, host=None, user=None, password=None, init_fun=None, ssl=True, reset=False):
```
Adding `*` to force callers to use keyword arguments.  
Perhaps `ssl` should be `use_ssl`?
What about the fact that SSL has evolved into TLS?

Reply:
*See comment in `Connection` class init definition.
Actually, mentioning `ssl` here was my mistake, per MySQL [doc](https://dev.mysql.com/doc/refman/5.7/en/encrypted-connections.html), it actually only uses TLS. 

>  TLS is sometimes referred to as SSL (Secure Sockets Layer) but MySQL does not actually use the SSL protocol for encrypted connections because its encryption is weak. 

I can update `ssl` to `use_tls`, however, this will accept 3 values: `True`, `False`, or a dictionary of cert paths e.g. `{'ca': 'path/to/ca', 'capath': 'path/to/ca/dir', 'cert': 'path/to/client/pub/key', 'key': 'path/to/client/priv/key', 'cipher': 'cipher1,cipher2'}` per PyMySQL doc [here](https://pymysql.readthedocs.io/en/latest/modules/connections.html?highlight=ssl).

Reply:
Addressed. Utilizing `use_ssl` as it seems that this is still the standard for SSL/TLS connection option. We can document that it is in fact utilizing the more secure TLS. As interim documentation, the logic is as follows: 

`True` = Require TLS
`False` = Use only non-TLS
`'Unset'` = Best available with TLS as primary (default)
(python dict) = Implies required TLS and a straight pass-thru to PyMySQL.